### PR TITLE
Add/action bar UI

### PR DESCRIPTION
**Pull request by roo2**
Moves the actionbar script to be loaded externally from widgets.wp.com.
Adds translations and variables to support the simple links in the "more" menu of the action bar https://github.com/Automattic/jetpack/issues/26005

#### Does this pull request change what data or activity we track or use?
Well it links to an external actionbar script on widgets.wp.com, but we don't use that to track any data/activity.

### Testing instructions
Apply 87180-code
To test how it would appear for wpcom users, you may have to hardcode the `isWpcom` variable

### CODE REVIEW

File: projects/packages/action-bar/src/class-action-bar.php [hunk 1]
```diff
@@ -38,8 +35,23 @@ public function enqueue_scripts() {
 				'more'         => esc_html__( 'More options', 'jetpack-action-bar' ),
 				'follow'       => esc_html__( 'Follow site', 'jetpack-action-bar' ),
 				'like'         => esc_html__( 'Like this post', 'jetpack-action-bar' ),
+				'report'       => esc_html__( 'Report this content', 'jetpack-action-bar' ),
+				'viewSite'     => esc_html__( 'View site in reader', 'jetpack-action-bar' ),
+				'manage'       => esc_html__( 'Manage subscriptions', 'jetpack-action-bar' ),
+				'readerUrl'    => $this->get_reader_url(),
+				'isWpcom'      => defined( 'IS_WPCOM' ) && IS_WPCOM,
```
Comment (roo2) [inline projects/packages/action-bar/src/class-action-bar.php:42]: isWpcom can be set to `true` to test the hidden menu options

**Comment by creativecoder** [comment]
Thanks for working on this!

This takes a kind of hybrid approach that runs the action bar js on the blog, but then loads it from widgets.wp.com. I'd caution against that for 2 reasons

- wpcom-block-editor does this and it has some problems: for example cache busting is problematic (the script is enqueued by Jetpack, so brower asset cache can only be busted by timing out or with a Jetpack release that changes the query string). This has bit us in the past when a user loads a version of the script that has a bug, and we release a fix, but they can't get the fix until the browser cache is busted.
- The dotorg plugin guidelines discourage this method of loading assets: "Calling third party CDNs for reasons other than font inclusions; all non-service related JavaScript and CSS must be included locally" (https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/#8-plugins-may-not-send-executable-code-via-third-party-systems)

Instead, I think we should choose either:

- Keep the UI code first party (executing on the blog page) and build the script within Jetpack, or
- Keep the UI code in wpcom and serve from within an iframe

I need to do more testing, but we may want to go the iframe approach for remote login tokens to work without being blocked. They require a new window to open with a wordpress.com login prompt to authenticate likes and follows. That login window is triggered within a widgets.wp.com iframe, and I think may be blocked by pop-up blockers if the user's click doesn't also originate from within that iframe. But I need to verify to be sure.

**Comment by roo2** [comment]
I added the UI and modal code back into jetpack, it was pretty smooth sailing, hopefully we've found the right home for everything.

**Comment by roo2** [comment]
closing in favour of https://github.com/Automattic/jetpack/pull/26246

**Pull request closed without merging**